### ISSUE TITLE
Compile error in test.rkt

### ISSUE THREAD

**Issue by samth** [lehitoskin/libtoxcore-racket#5]
```
raco setup: --- summary of errors ---
raco setup: error: during making for <pkgs>/libtoxcore-racket/test
raco setup:   libtoxcore-racket/test/test.rkt:60:1: set-name: unbound identifier in module
raco setup:     in: set-name
raco setup:     compiling: <pkgs>/libtoxcore-racket/test/test.rkt
```

**Comment by lehitoskin** [comment]
The test file is pretty out of date. I haven't updated it since I messed with the API.

**Comment by lehitoskin** [comment]
Should be fixed with 689d186b9f7150d6ebbbd8e187f9b52373be119f
